### PR TITLE
Added redirect mode.

### PR DESCRIPTION
**Pull request by mamurshe**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: samples/AzureKeyVault/apiProperties.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
           "properties": {
             "IsFirstParty": "False"
           },
+		  "redirectMode": "Global",
```
Comment (ShyamSunder90) [inline samples/AzureKeyVault/apiProperties.json]: Indentation looks incorrect

Comment (mamurshe) [reply samples/AzureKeyVault/apiProperties.json]: Fixed.

File: samples/AzureKeyVault/apiProperties.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
           "properties": {
             "IsFirstParty": "False"
           },
+          "redirectMode": "Global",
           "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```
Comment (dmkuby) [inline samples/AzureKeyVault/apiProperties.json]: Do we need to specify both 'redirectMode' and 'redirectUrl'?

Comment (mamurshe) [reply samples/AzureKeyVault/apiProperties.json]: Yes. Note the same with Facebook and O365 connector in this repo.

**Pull request merged**